src/woo_publications/conf/base.py

```diff
@@ -80,6 +81,38 @@
     "https://identifier.overheid.nl/tooi/def/thes/kern/c_816e508d"
 )
 
+# Image field validator settings
+ALLOWED_IMG_EXTENSIONS = config(
+    "ALLOWED_IMG_EXTENSIONS",
+    default=[
+        "jpg",
+        "jpeg",
+        "png",
+        "gif",
+        "webp",
+    ],
+    help_text=_("The allowed image extensions that we support."),
+)
```

Comment:
iirc there's a `group` or `section` parameter as well to group these together. A heading like "Image upload settings" would be good.